### PR TITLE
Closes #287 by adding the first-timers-friendly badge to README

### LINKED ISSUE #287: Add first-timers-friendly badge to README
[first-timers-only](https://www.firsttimersonly.com/)

```
[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/)
```

---

> ✅️ By submitting this issue, I have verified the following

- [x] Checked to [see if the issue has already been discussed before](https://github.com/techqueria/website/issues) 🤔️
- [x] Made sure enough details were provided if proposing a new feature 🔍️

### PR DESCRIPTION
**Pull request by sonali9696**
Closes #287 

This PR adds the first-timers-friendly badge to README
---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite.svg?type=small)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite?ref=badge_small)
 
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/)
```
Comment (fvcproductions) [inline README.md]: @sonali9696 can we remove `?style=flat-square` so it matches format of other badges and also move it to be above FOSSA status?

**Comment by sonali9696** [comment]
Hi @fvcproductions, it would be great if you could review the PR and let me of any changes to be done to proceed with it.

File: README.md (hunk 1 above)
Comment (sonali9696) [reply README.md]: Made the required changes

**Comment by sonali9696** [comment]
I have resolved the placement and format issue and have merged the conflicts as well due to multiple contributors changing the README file.

**Comment by fvcproductions** [comment]
Thanks @sonali9696 - much appreciated! 💯

**Comment by sonali9696** [comment]
Loved working on the issues and would look forward to making more contributions to your website. :)

**Pull request merged**